langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -275,6 +389,12 @@ public isolated function fromCodePointInt(int codePoint) returns Char|error = @j
 # Adds sufficient `padChar` characters at the start of `str` to make its length be `len`.
 # If the length of `str` is >= `len`, returns `str`.
 #
+# ```ballerina
+# // Providing "0" as the custom padding char
```

Comment:
Shall we move the comment closer to the call?

Reply:
Applicable elsewhere too.

Reply:
Sure.